engine/audio/src/audio.cpp

```diff
@@ -225,7 +224,7 @@ int Audio::adjustAttribute(qreal fraction, int attributeId)
 
 void Audio::slotEndOfStream()
 {
-    if (!stopped())
+    if (!isStopped())
```

Comment:
This change is done because there is another method named `stopped` with a different parameter list. One can use lambda's, but renaming to `isStopped()` is cleaner IMO. There is already an `isPaused()` and `isRunning()`.